maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -87,6 +87,12 @@ class TestCommand : Callable<Int> {
     )
     private var debugOutput: String? = null
 
+    @Option(
+        names = ["--flatten-debug-output"],
+        description = ["Don't create subfolders or timestamps for each run. Useful for CI."]
```

Comment:
Let's improve this description, something like:
```
All file outputs from the test case are created in the folder without subfolders or timestamps for each run. It can be used with --debug-output. Useful for CI
```

Reply:
Far prefer your wording, switching it to a positive stance.